### ISSUE TITLE
[Feature request] Allow users to subscribe without logging in

### ISSUE THREAD

**Issue by MegaArthur** [ajh3/NoSurfForReddit#17]
It'd be really nice if you could subscribe (add a sub to "your subreddits" without needing a Reddit account. Slide, another FOSS Reddit client does just that.

**Comment by MegaArthur** [comment]
Yep, pretty much. I do not wish for Reddit to profile my actions and I usually just browse one or two subreddits related to privacy.

**Comment by ajh3** [comment]
Like you, I am very concerned with digital privacy. Still, your best bet is just to create an account. Reddit nevertheless profiles your actions by tracking your IP address, so you won't escape tracking by browsing anonymously. Creating an account only requires an e-mail address (and that may even be optional), so it truly won't expose you to any additional data gathering. The only additional threat of having an account is if you voluntarily choose to post information; that's by far the most serious way of losing privacy on Reddit. If you don't post anything, then it really does not make any difference from a privacy perspective whether you use an account or not.

On that basis, I probably won't implement this feature, as it would require a significant effort and there are too many other priorities right now.

**Comment by MegaArthur** [comment]
Fair point.
You can close the issue if you wish to, or just leave it hanging about, open.
